plugins/polio/tasks/api/refresh_lqas_data.py

```diff
@@ -136,10 +139,21 @@ def create(self, request):
         return Response({"task": TaskSerializer(instance=task).data})
 
     def refresh_lqas_data(self, country_id=None, task_id=None):
+        try:
+            pipeline_config = get_object_or_404(Config, slug="lqas-pipeline-config")
+        except:
+            logger.exception("Could not fetch openhexa config")
+            return ERRORED
+        lqas_pipeline_version = pipeline_config.content["lqas_pipeline_version"]
+        oh_pipeline_target = pipeline_config.content["oh_pipeline_target"]
+        lqas_pipeline = pipeline_config.content["lqas_pipeline"]
+        openhexa_url = pipeline_config.content["openhexa_url"]
+        openhexa_token = pipeline_config.content["openhexa_token"]
+
```

Comment:
I think this could crash if one of these keys is not present in the config.content JSON thing.
Maybe put a big try catch around it or something as I don't think there is anywhere where the structure of this config is verified. 

Alternatively and also for documentation purpose, you could use the jsonschema lib to verify this structure (See `iaso/tests/api/workflows/test_workflows.py` for an full example)